File: sorting/pySolutions/01-bubble_sort-easy.py
def bubble_sort(list):
    swapped = True
    while swapped:
        swapped = False
        for i in range(len(list) - 1):
            if list[i] > list[i+1]:
                temp = list[i]
                list[i] = list[i+1]
                list[i+1] = temp
                swapped = True
    return list


print(bubble_sort([2, 9, 3, 3, 4, 5, 76, 96, 453, 46, 8, 6, ]))

File: sorting/jsSolutions/01-bubbleSort-easy.js
const bubbleSort = (arr) => {
  let count = 0, swapped;
  do {
    swapped = false
    for (let i = 0; i < arr.length - count; i++) {
      if (arr[i] > arr[i + 1]) {
        [arr[i], arr[i + 1]] = [arr[i + 1], arr[i]]
        swapped = true
      }
    }
  } while (swapped)
  return arr
}

console.log(bubbleSort([2, 9, 3, 3, 4, 5, 76, 96, 453, 46, 8, 6,]))


